tasks/components/layout/WardRoomList.tsx

```diff
@@ -68,10 +68,12 @@ export const WardRoomList = ({
         hasError={isError}
       >
         {rooms && rooms.map(room => (
+          room.beds.length > 0 ? (
             <RoomOverview
               key={room.id}
               room={room}
             />
+          ) : ''
```

Comment:
use a filter instead
```suggestion
        {rooms && rooms.filter(room => room.beds.length > 0).map(room => ( 
            <RoomOverview
              key={room.id}
              room={room}
            />
```